.profile

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+
+# New Relic
+service_name="datagov-harvest-secrets"
```

Comment:
i figured we could just bind the harvest secrets service to this app instead of creating a separate service